src/main/resources/templates/activities.html

```diff
@@ -49,6 +51,8 @@
             <td>{inject:prettyTime.format(activityEntry.created)} ({activityEntry.created})</td>
             <td>{activityEntry.repository}</td>
             <td><a href="{activityEntry.url}">{activityEntry.url}</a></td>
+            <td>{activityEntry.labels}</td>
```

Comment:
The table looks a bit off with labels, especially when you run it in non-dev mode, I didn't add the title of PR/Issue for the same reason.

Please drop label line.

Info about labels could be reused as tooltip on table cell for URL. Or maybe just drop it completely ... up to you.

Reply:
Will remove the column but keep the labels for future usages